packages/datadog-plugin-jest/src/index.js

```diff
@@ -256,5 +416,15 @@ module.exports = [
       this.unwrap(JsdomEnvironment.prototype, 'teardown')
       JsdomEnvironment.prototype.handleTestEvent = JsdomEnvironment.prototype.handleTestEvent._dd_original
     }
+  },
+  {
+    name: 'jest-jasmine2',
+    versions: ['>=24.8.0'],
+    file: 'build/jasmineAsyncInstall.js',
+    patch: function (jasmineAsyncInstallExport, tracer) {
```

Comment:
This should also have a corresponding `unpatch`.

Reply:
how would you do it? We're returning a wrapped function instead of the original function, not using `this.wrap`. Is there an example in the codebase for "unwrapping" a returned function?

Reply:
The instrumenter support this out of the box. The [connect](https://github.com/DataDog/dd-trace-js/blob/34cfecdf54087f4a4fcdd772588b6bc172351059/packages/datadog-plugin-connect/src/index.js#L103-L109) plugin is a good example.

Reply:
nice, thanks 👍